PAPI_FAQ.html

```diff
@@ -16,20 +16,28 @@ <h1>PAPI FAQ</h1><h2>General Questions (FAQ)</h2>
 <p><a href="mailto:ptools-perfapi@icl.utk.edu">ptools-perfapi@icl.utk.edu</a></p>
 <p>Please note, this is a moderated group and if you are not subscribed to it, there may be a delay till the moderators get to your message and approve it.</p><strong>How do I install the PAPI library?</strong>
 <p>Please see INSTALL.txt in the papi root directory.</p><strong>Where do I go for help?</strong>
-<p>First, read this document thoroughly. Then consult the PAPI Home Page at <a href="../">http://icl.utk.edu/papi</a>. If that doesn't help, then search the archives as mentioned below. If that fails, then send mail to one of the two mailing lists, the users group at <a href="mailto:ptools-perfapi@icl.utk.edu">ptools-perfapi@icl.utk.edu</a>&nbsp;or the PAPI-developers group <a href="mailto:perfapi-devel@icl.utk.edu">perfapi-devel@icl.utk.edu</a>. The former is a group for general announcements, questions and miscellaneous topics. The latter is is a discussion group for the developers of PAPI and it receives all CVS update messages. (which can be a significant amount of mail!)</p><strong>What are the mailing lists and how do I subscribe?</strong>
+<p>First, read this document thoroughly. Then consult the PAPI Home Page at <a href="http://icl.utk.edu/papi">http://icl.utk.edu/papi</a>. If that doesn't help, then search the archives as mentioned below. If that fails, then send mail to one of the two mailing lists, the users group at <a href="mailto:ptools-perfapi@icl.utk.edu">ptools-perfapi@icl.utk.edu</a>&nbsp;or the PAPI-developers group <a href="mailto:perfapi-devel@icl.utk.edu">perfapi-devel@icl.utk.edu</a>. The former is a group for general announcements, questions and miscellaneous topics. The latter is is a discussion group for the developers of PAPI and it receives all CVS update messages. (which can be a significant amount of mail!)</p><strong>What are the mailing lists and how do I subscribe?</strong>
```

Comment:
I think adding a direct link to the PAPI wiki documentation here would be a good idea.

It could possibly be structured as follows:
**First, read this document and the [PAPI documentation ](https://github.com/icl-utk-edu/papi/wiki)thoroughly.**